code/modules/mob/living/carbon/human/human.dm

```diff
@@ -903,4 +908,19 @@
 	if(s_store)
 		add_screams(s_store.alternate_screams)
 	if(wear_id)
-		add_screams(wear_id.alternate_screams)
\ No newline at end of file
+		add_screams(wear_id.alternate_screams)
+
+//I'm not even sure where this function went in our code... It's in RR's PR so ???
+// /mob/living/carbon/human/can_use_hands()
+// 	if(!..())
+// 		return 0
+// 	if(!has_active_hand())
+// 		return 0
+// 	return 1
+
+
+/mob/living/carbon/human/revive()
+	for(var/obj/item/organ/limb/L in organs)
+		L.change_organ(L.status)
```

Comment:
Oh, that is actually a useless proc call since I reworked how RR's limbs work.
I'll make it properly regenerate your limbs later